src/core/src/FabricGeometry.cpp

```diff
@@ -203,24 +195,27 @@ void FabricGeometry::reset() {
     auto extentFabric = srw.getAttributeWr<pxr::GfRange3d>(_pathFabric, FabricTokens::extent);
     auto worldExtentFabric = srw.getAttributeWr<pxr::GfRange3d>(_pathFabric, FabricTokens::_worldExtent);
     auto worldVisibilityFabric = srw.getAttributeWr<bool>(_pathFabric, FabricTokens::_worldVisibility);
-    auto tilesetIdFabric = srw.getAttributeWr<int64_t>(_pathFabric, FabricTokens::_cesium_tilesetId);
-    auto tileIdFabric = srw.getAttributeWr<int64_t>(_pathFabric, FabricTokens::_cesium_tileId);
     auto localToEcefTransformFabric = srw.getAttributeWr<pxr::GfMatrix4d>(_pathFabric, FabricTokens::_cesium_localToEcefTransform);
     auto worldPositionFabric = srw.getAttributeWr<pxr::GfVec3d>(_pathFabric, FabricTokens::_worldPosition);
     auto worldOrientationFabric = srw.getAttributeWr<pxr::GfQuatf>(_pathFabric, FabricTokens::_worldOrientation);
     auto worldScaleFabric = srw.getAttributeWr<pxr::GfVec3f>(_pathFabric, FabricTokens::_worldScale);
+    auto displayColorFabric = srw.getArrayAttributeWr<pxr::GfVec3f>(_pathFabric, FabricTokens::primvars_displayColor);
+    auto displayOpacityFabric = srw.getArrayAttributeWr<float>(_pathFabric, FabricTokens::primvars_displayOpacity);
     // clang-format on
 
     *extentFabric = DEFAULT_EXTENT;
     *worldExtentFabric = DEFAULT_EXTENT;
-    *worldVisibilityFabric = false;
-    *tilesetIdFabric = -1;
-    *tileIdFabric = -1;
+    *worldVisibilityFabric = DEFAULT_VISIBILITY;
     *localToEcefTransformFabric = DEFAULT_MATRIX;
     *worldPositionFabric = DEFAULT_POSITION;
     *worldOrientationFabric = DEFAULT_ORIENTATION;
     *worldScaleFabric = DEFAULT_SCALE;
+    displayColorFabric[0] = DEFAULT_VERTEX_COLOR;
+    displayOpacityFabric[0] = DEFAULT_VERTEX_OPACITY;
 
+    FabricUtil::setTilesetId(_pathFabric, -1);
```

Comment:
NIT: Might be worthwhile to explain in a comment or better yet, function documentation for `setTilesetId` as to what `-1` signifies.